ush/wafs_grib2_0p25_blending.sh

```diff
@@ -0,0 +1,145 @@
+#              - Extend waiting time window from 15 to 25 minutes
+################################################################################
+
+set -x
+
+fhr=$1
+
+mkdir -p $DATA/$fhr
+cd "${DATA}/${fhr}" || err_exit "FATAL ERROR: Could not 'cd ${DATA}/${fhr}'; ABORT!"
+
+###############################################
+# Specify Timeout Behavior for WAFS blending
+###############################################
+# SLEEP_TIME - Amount of time (secs) to wait for a input file before exiting
+# SLEEP_INT  - Amount of time (secs) to wait between checking for input files
+SLEEP_TIME=${SLEEP_TIME:-1500}
+SLEEP_INT=${SLEEP_INT:-10}
+SLEEP_LOOP_MAX=$((SLEEP_TIME / SLEEP_INT))
+
+##########################
+# look for UK WAFS data.
+##########################
+MISSING_UK_WAFS="NO"
+
+for ((ic = 1; ic <= SLEEP_LOOP_MAX; ic++)); do
+    # Three(3) unblended UK files for each cycle+fhour: icing, turb, cb
+    ukfiles=$(find "${COMINuk}" -name "egrr_wafshzds_unblended_*_0p25_${PDY:0:4}-${PDY:4:2}-${PDY:6:2}T${cyc}:00Z_t${fhr}.grib2" | wc -l)
+    if ((ukfiles >= 3)); then
+        echo "Found all 3 UK WAFS GRIB2 files, continue ..."
+        break
+    fi
+
+    if ((ic == SLEEP_LOOP_MAX)); then
+        products="cb ice turb"
+        for prod in ${products}; do
+            ukfile="egrr_wafshzds_unblended_${prod}_0p25_${PDY:0:4}-${PDY:4:2}-${PDY:6:2}T${cyc}:00Z_t${fhr}.grib2"
+            if [[ ! -f "${COMINuk}/${ukfile}" ]]; then
+                echo "WARNING: UK WAFS GRIB2 file '${ukfile}' not found after waiting over ${SLEEP_TIME} seconds"
+                echo "Missing ${COMINuk}/${ukfile}" >> ../missing_uk_files.$fhr
+            fi
+        done
+        echo "WARNING: UK WAFS GRIB2 unblended data is not completely available, no blending"
+        MISSING_UK_WAFS="YES"
+        break
+    else
+        sleep "${SLEEP_INT}"
+    fi
+done
+
+##########################
+# look for US WAFS data.
+##########################
+MISSING_US_WAFS="NO"
+
+if [[ ! -f "${COMINus}/WAFS_0p25_unblended_${PDY}${cyc}f${fhr}.grib2.idx" ]]; then
+    # Blending ecflow run is triggered by the finishing of the upstream JWAFS_GRIB2_0P25 f048
+    #  - In real time ecflow, blending starts at T+4:30, unblended upstream products are all available.
+    #    It doesn't need to sleep/wait
+    #  - In non-real time ecflow, blending has no time tigger, unblended f048 is not guaranteed to be the last finished one.
+    #    It needs to sleep/wait. All upstream jobs can be finished within 60 seconds when f048 is finished.
+    #
+    # Standalone blending doesn't need to sleep/wait since unblended upstream grib2_0p25 runs standalone first.
+    sleep 60
+    if [[ ! -f "${COMINus}/WAFS_0p25_unblended_${PDY}${cyc}f${fhr}.grib2.idx" ]]; then
+	echo "WARNING: missing US unblended data - ${COMINus}/WAFS_0p25_unblended_${PDY}${cyc}f${fhr}.grib2"
+	echo "Missing ${COMINus}/WAFS_0p25_unblended_${PDY}${cyc}f${fhr}.grib2" > ../missing_us_file.$fhr
```

Comment:
```suggestion
	echo "Missing ${COMINus}/WAFS_0p25_unblended_${PDY}${cyc}f${fhr}.grib2" > ../missing_us_files.$fhr
```
Name the missing us file the same way the uk and no blending are; `files` instead of `file`